dvc/command/pipeline.py

```diff
@@ -162,13 +162,18 @@ def run(self):
 
 class CmdPipelineList(CmdBase):
     def run(self):
-        import networkx
+        import networkx as nx
 
         pipelines = self.repo.pipelines
         for p in pipelines:
-            stages = networkx.get_node_attributes(p, "stage")
-            for stage in stages:
-                logger.info(stage)
+            stages = nx.get_node_attributes(p, "stage")
+            for node in p:
+                stage = stages[node]
+                # if not stage.locked:
```

Comment:
Please don't leave commented-out code like that. Looks like you don't need it, right? If so - remove it.